### ISSUE TITLE
Apply consistent code formatting

### ISSUE THREAD

**Issue by mbabker** [joomla-projects/cookiescript#5]
It is impossible to review the JS file because the code formatting within it is inconsistent.  A coding standard should be applied and followed to ensure anyone following this repo or contributing to its development can easily jump in and get involved.

**Comment by jeckodevelopment** [comment]
Thank you for the suggestion @mbabker 
I'll try to review it

**Comment by Alcaeusd** [comment]
More or less, this is the format CIVIC itself uses for the script. Although, yes we can try to review it.

**Comment by mbabker** [comment]
When you have inconsistent indenting, it makes it pretty difficult to review the code, and items like #3 are easy to slip in because it's hard to follow what thing is where.

This is the same file run through my IDE's autoformatter:

```js
var config = {
    apiKey: '[INSERT API KEY HERE]',
    product: 'PRO_MULTISITE',
    initialState: "NOTIFY",
    consentCookieExpiry: 90,
    notifyOnce: false,
    position: "LEFT",
    theme: "LIGHT",
    layout: "slideout",
    branding: {
        fontColor: "#fff",
        fontSizeTitle: "1.1em",
        fontSizeIntro: "1em",
        fontSizeHeaders: "1em",
        fontSize: "0.95em",
        backgroundColor: '#1a3867',
        toggleText: '#000',
        toggleColor: '#ccc',
        toggleBackground: '#fff',
        buttonIcon: null,
        buttonIconWidth: "64px",
        buttonIconHeight: "64px",
        removeIcon: false,
        removeAbout: true
    },
    text: {
        title: 'This website uses cookies to remember users and understand ways to enhance their experience.',
        intro: 'Some cookies are necessary, while other cookies help us improve your experience, while you are navigating through our website. For further information, please visit our <a href="https://www.joomla.org/privacy-policy.html" target="_blank">Privacy Policy</a>.',
        acceptRecommended: 'Accept Cookies',
        necessaryTitle: 'Manage Cookie Preferences',
        necessaryDescription: '<strong>Strictly Necessary Cookies</strong> are essential for our website to function properly, for instance authenticating logins or serving files, crucial for the core functionality. You can only disable necessary cookies via browser settings but if you do, our website might not be properly functional for your needs.',
        notifyDescription: 'We use cookies to optimize site functionality and give you the best possible experience. Learn more.'
    },
    necessaryCookies: ['YOUR_SESSION_COOKIE'],
    optionalCookies: [
        {
            name: 'performance',
            label: 'Performance Cookies',
            description: 'Performance cookies help us to improve our website by collecting and reporting information, on its usage.',
            cookies: ['_dc_gtm_UA*', '_ga', '_gid'],
            onAccept: function () {
                (function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
                new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
                j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
                '//www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
                })(window,document,'script','dataLayer','GTM-5GST4C');
            },
            onRevoke: function () {
                // Disable Google Analytics
                window['ga-disable-UA-XXXXXX-XX'] = true;
                // End Google Analytics
                CookieControl.delete('_dc_gtm_UA-XXXXXX-XX');
                CookieControl.delete('_ga');
                CookieControl.delete('_gid');
            },
            recommendedState: true,
            lawfulBasis: 'legitimate interest'
        },
        {
            name: 'advertising',
            label: 'Advertising Cookies',
            description: 'Advertising cookies help you see some ads based on your preferences. Joomla! serves or hosts ads as they are one of its major financial support.',
            cookies: ['IDE'],
            onAccept: function () {
            },
            onRevoke: function () {
                CookieControl.delete('IDE');
            }
        },
        {
            name: 'accept',
            label: ' ',
            description: '<a href="" onclick="CookieControl.hide();">Continue to site</a>',
            toggleType: 'checkbox'
        }
    ],
    statement: {},
};
window.addEventListener("load", function () {
    CookieControl.load(config);
    var removex = document.getElementById('ccc-notify-dismiss');
    if (removex) {
        dismissIcon.remove(); // remove the click function if element exists
    }

    //if type is closed the cookie notice is not accepted. if it is notify, then it is accepted.
    if (JSON.parse(CookieControl.getCookie('CookieControl')).initialState.type == 'closed' && JSON.parse(CookieControl.getCookie('CookieControl')).initialState.type != 'undefined') {
        CookieControl.open();
    } else {
        console.log('required refresh');
    }
});
```

Consistent indention makes it easy to see where one thing stops and another starts.

**Comment by Alcaeusd** [comment]
Thanks @mbabker !!

**Comment by jeckodevelopment** [comment]
Should be ok now.
Please reopen if needed
